src/SigFox.cpp

```diff
@@ -122,6 +122,12 @@ int SIGFOXClass::begin(SPIClass& spi, int reset, int poweron, int interrupt, int
 int SIGFOXClass::send(unsigned char mess[], int len, bool rx)
 {
   if (len == 0) return -1;
+
+  if (rx == false && len == 1 && mess[0] < 2) {
+    //we can use send_bit command
```

Comment:
@nicolsc @facchinm will this change behaviour if the remote side is expecting a 0 or 1 in a single byte?